.proto/kannon/admin/apiv1/hz.proto

```
@@ -0,0 +1,11 @@
+package pkg.kannon.admin.apiv1;
```

Buf (1.55.1)

Files must have a syntax explicitly specified. If no syntax is specified, the file defaults to "proto2".

(SYNTAX_SPECIFIED)